doc/client-setup.md

```diff
@@ -106,9 +106,33 @@ let externalLink = `<a href="https://external-site.com/external/path" target="_t
 ## Requesting Host Actions
 
 There are common requests a client application will want to make of a host application and we strive
-to provide nice default APIs for these. Currently there is only a single api implemented:
+to provide nice default APIs for these. 
+
+Currently there are two implemented:
+
 [`requestNotification`](../classes/client.client-1.html#requestnotification), which asks the host
-app to send a notification message to the user. More will be added in future releases.
+app to send a notification message to the user.
+
+[`requestModal`](../classes/client.client-1.html#requestModal), which asks the host
+app to launch a modal identified by a given ID, also accepts initial setup data specific to that modal.
+
+A client application may request a modal on the host like so, this fires a 'modalRequest' event for the host to listen for:
+```typescript
+ifcClient.requestModal({
+  modalType: 'idOfTheModalToDisplay',
+  modalData: {
+    id: '1234567890',
+    userList : [{id: '2345678901', name: 'User1'}]
+});
+```
+The 'modalRequest' event is received by the host application.
+Below is an example (Ember).
+```javascript
+modalRequest(event) {
+  // Do something with the event here
+},
+this.element.addEventListener('modalRequest', this.modalRequest.bind(this));
+```
```

Comment:
This host example is pulled from web-dir, but only makes sense with a lot of implicit context from that app that may not carry over in general.  I'd just summarize that the `frame-router` element will emit a custom event with the `ModalRequest` object in the detail property.